components/language-chooser/common/find-language/scriptSamples.ts

```diff
@@ -1,11 +1,22 @@
 export const scriptSamples: { [key: string]: string } = {
+  // Scripts which frequently show up as options for languages with multiple scripts
   latn: "A B C D",
   arab: "ا ب ت ث",
-  deva: "अ आ इ ई",
+  deva: "क ख ग घ",
```

Comment:
changed to be consistent with the other Indian script samples (most of the new samples are for the consonant sounds ka, kha, ga, gha)